util/edns0_test.go

```diff
@@ -90,7 +90,6 @@ var _ = Describe("EDNS0 utils", func() {
 			opt.Hdr.Name = "."
 			opt.Hdr.Rrtype = dns.TypeOPT
 			eso := new(dns.EDNS0_SUBNET)
-			eso.Code = dns.EDNS0SUBNET
```

Comment:
Is this intentional?

Reply:
Yes since `Code` is completely irrelevant if `.Option()` is used. 😉